example/tests/test_serializers.py

```diff
@@ -30,6 +32,32 @@ def setUp(self):
                 Author.objects.create(name=name, email='{}@example.org'.format(name))
             )
 
+    def test_forward_relationship_not_loaded_when_not_included(self):
+        MockRequest = namedtuple('Request', ['query_params'])
+        request_without_includes = MockRequest({})
+
+        class BlogSerializer(ModelSerializer):
+            class Meta:
+                model = Blog
+                fields = '__all__'
+
+            def to_representation(self, instance):
+                raise Exception('to_representation of BlogSerializer was called')
+
+        class EntrySerializer(ModelSerializer):
+            blog = BlogSerializer()
+
+            class Meta:
+                model = Entry
+                fields = '__all__'
+
+            included_serializers = {
+                'blog': BlogSerializer,
+            }
+
+        serializer = EntrySerializer(context={'request': request_without_includes})
+        serializer.to_representation(self.entry)
```

Comment:
I think it would also be good to assert the result.

Reply:
I agree. I added another test to validate the result.